x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/management/trial_license_complete_tier/upgrade_review_prebuilt_rules.ts

```diff
@@ -711,5 +711,510 @@ export default ({ getService }: FtrProviderContext): void => {
         });
       });
     });
+
+    describe(`scalar array fields`, () => {
+      const getRuleAssetSavedObjects = () => [
+        createRuleAssetSavedObject({
+          rule_id: 'rule-1',
+          version: 1,
+          tags: ['one', 'two', 'three'],
+        }),
+      ];
+
+      describe("when rule field doesn't have an update and has no custom value", () => {
+        it('should not show in the upgrade/_review API response', async () => {
+          // Install base prebuilt detection rule
+          await createHistoricalPrebuiltRuleAssetSavedObjects(es, getRuleAssetSavedObjects());
+          await installPrebuiltRules(es, supertest);
+
+          // Increment the version of the installed rule, do NOT update the related scalar array field, and create the new rule assets
+          const updatedRuleAssetSavedObjects = [
+            createRuleAssetSavedObject({
+              rule_id: 'rule-1',
+              tags: ['one', 'three', 'two'],
+              version: 2,
+            }),
+          ];
+          await createHistoricalPrebuiltRuleAssetSavedObjects(es, updatedRuleAssetSavedObjects);
+
+          // Call the upgrade review prebuilt rules endpoint and check that there is 1 rule eligable for update but scalar array field is NOT returned
+          const reviewResponse = await reviewPrebuiltRulesToUpgrade(supertest);
+          expect(reviewResponse.rules[0].diff.fields).toEqual({
+            version: {
+              base_version: 1,
+              current_version: 1,
+              target_version: 2,
+              merged_version: 2,
+              diff_outcome: ThreeWayDiffOutcome.StockValueCanUpdate,
+              merge_outcome: ThreeWayMergeOutcome.Target,
+              has_conflict: false,
+              has_update: true,
+            },
+          });
+          expect(reviewResponse.rules[0].diff.has_conflict).toBe(false);
+          expect(reviewResponse.stats.num_rules_to_upgrade_total).toBe(1);
+        });
+      });
+
+      describe("when rule field doesn't have an update but has a custom value", () => {
+        it('should not show in the upgrade/_review API response', async () => {
```

Comment:
It <b>should</b> in the API response, even though the diff outcome is`ThreeWayDiffOutcome.CustomizedValueNoUpdate`. This is part of the new logic we introduced in [this PR](https://github.com/elastic/kibana/pull/184889): only outcomes of `StockValueNoUpdate` don't get returned in the reponse now.

It looks that we have to correct the `it` descriptions for the other test cases, since they also say that the field diff shouldn't be a part of the response, although they should.